src/components/Button/index.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { shallow } from 'enzyme'
+import Button from '../Button';
+
+
+describe('Button', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(<Button />);
+  });
+
+  test('renders without crashing', () => {
```

Comment:
This duplicates the line from the `beforeEach` block. I feel like it's safe to remove it since it is implied that it will render without crashing with any test this describe block.